test/CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ set(RGL_TEST_FILES
 if (RGL_BUILD_ROS2_EXTENSION)
     list(APPEND RGL_TEST_FILES
         src/graph/nodes/Ros2PublishRadarScanNodeTest.cpp
+        src/scene/rcsAngleDistributionTest.cpp
```

Comment:
Now I saw ros2 publishing in test which I assume is for testing. Can stay for now, maybe will be updated later.

Reply:
This is ROS-2 only, because currently it's the most convenient way to visualize results, as our internal Visualizer is less trustworthy and feature-rich as RViz2.